src/app/personside/infotabs/ytelser/sykepenger/utbetalingerpåvent/utledUtbetalingPåVentÅrsak.test.ts

```diff
@@ -0,0 +1,159 @@
+import { utledUtbetalingPåVentÅrsak } from './utledUtbetalingerPåVentÅrsak';
+import { UtbetalingPåVent } from '../../../../../../models/ytelse/ytelse-utbetalinger';
```

Comment:
Yassss sjukt bra tester! ❤️  ❤️ 

Protip: Rename denne filen fra `utledUtbetalingPåVentÅrsak.test.ts` til `utledUtbetalingerPåVentÅrsak.test.ts` (utbetaling vs utbetalingER)

Så går du inn i en av dem og trykker Ctrl + alt + T, da kan du hoppe mellom filen som tester og den som blir testet(TDD FTW). Fin måte å sjekke om det finnes testfiler for en fil, samt navigere :) MEN Navnene må være identiske for at IntelliJ skal forstå hvilken de er knyttet til

Reply:
Bra catch!